package.json

```diff
@@ -310,6 +369,47 @@
         "language": "sas",
         "path": "./snippets/function-snippets.json"
       }
+    ],
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "sas-view-container",
+          "title": "SAS",
+          "icon": "icons/sas-svg.svg"
+        }
+      ]
+    },
+    "views": {
+      "sas-view-container": [
+        {
+          "id": "sas-content-navigator",
+          "name": "Explorer",
+          "when": "SAS.authorized && SAS.connectionMethod == rest"
+        },
+        {
+          "id": "sas-content-get-started",
+          "name": "Get Started",
+          "when": "!SAS.authorized && SAS.connectionMethod == rest"
+        },
+        {
+          "id": "sas-content-invalid-connection",
+          "name": "Get Started",
+          "when": "!SAS.authorized && SAS.connectionMethod != rest"
+        }
+      ]
+    },
+    "viewsWelcome": [
+      {
+        "view": "sas-content-get-started",
+        "when": "!SAS.authorized && SAS.connectionMethod == rest",
+        "contents": "You have not yet signed in to SAS.\n[Sign in](command:SAS.authorize)",
+        "enablement": "!SAS.authorizing"
+      },
+      {
+        "view": "sas-content-invalid-connection",
+        "when": "SAS.connectionMethod != rest",
+        "contents": "Your connection does not support SAS content navigation."
+      }
```

Comment:
@skyeaussie @crlewi Just want to highlight these two bits of text. Will you confirm these are okay or provide updated content

Reply:
BTW., let's put UI text into the nls file whenever we can

Reply:
Good point. I've a kind of related question. Is there anyway to tap into package.nls.json within our extensions code? I briefly looked into it, but didn't find what I was looking for. It'd be ideal if _all_ of our text came from package.nls.json (imo)